### ISSUE TITLE
time for a new release?

### ISSUE THREAD

**Issue by MartinNowak** [jacob-carlborg/dstep#155]
![dstep_0 2 1](https://user-images.githubusercontent.com/288976/27682648-b5009f4c-5cc3-11e7-8b80-9ff235ba1b46.png)
Almost 200 commits in the past 1.5 years, not to mention that it no longer compiles with anything newer than 2.071.2.

**Comment by jacob-carlborg** [comment]
Yes, it's time for a new release. It does compile with at least 2.074.0. I've haven't updated the travis configuration. I've been working for some time now on a couple of things that will lead up to a new release, hopefully pretty soon.

**Comment by jacob-carlborg** [comment]
Compiles with 2.075.0-b1 as well.

**Comment by MartinNowak** [comment]
> It does compile with at least 2.074.0.

Well neither 0.2.2 nor 0.2.1 do ;), `__va_argsave` is no longer found in Mambo/Tango, but looks like you got rid of that dependency :).

**Comment by jacob-carlborg** [comment]
Well, try master 😃. Travis CI is now setup to test using the latest DMD, LDC, DMD beta and DMD nightly. It also uses the scheduled jobs/cron feature of Travis CI to build everyday to make sure nothing breaks even if nothing has been pushed.

https://github.com/jacob-carlborg/dstep/blob/master/.travis.yml

**Comment by skl131313** [comment]
It'd help if you setup travis/appveyor to upload the binaries they make. The last released binary is from 2016 and there is none for Windows. Clang isn't really a small dependency, especially for Windows where I think you'd have to build it yourself. It'd be nice to have bianries, especially since it shouldn't be that much difficult to setup in the CI.

**Comment by jacob-carlborg** [comment]
I've been working on that recently, in the `release` branch [1]. As a dependency on that I had to build LLVM myself since the official releases are missing the `libclang.a` library. This turned out to take quite a while and caused some headaches. I had to start using CircleCI because it takes too long time to build Lfor Travis CI. Which means I had to learn a new CI platform. But since this is done now [2] I can get back to this again.

So what I need to do now is to somehow indicate to Dub that a release build should be performed, which will look slightly differently from a regular build. It should also run the end-to-end tests using the release build.

[1] https://github.com/jacob-carlborg/dstep/tree/release
[2] https://github.com/jacob-carlborg/llvm-svn/releases/tag/dstep-5.0.1
